lisa/tests/base.py

```diff
@@ -348,13 +348,9 @@ def to_dir(self, res_dir):
         """
         super().to_path(self._filepath(res_dir))
 
-class RTATestBundle(TestBundle):
+class RTATestBundle(TestBundle, abc.ABC):
```

Comment:
AFAICT That's not needed - I removed some of those to make the class diagrams less confusing, and `TestBundle` is likely to stay abstract for the foreseeable future.

Reply:
+1 for class diagram